static/app/components/events/meta/metaProxy.tsx

```diff
@@ -22,7 +24,11 @@ export class MetaProxy {
     this.local = local;
   }
 
-  get<T extends {}>(obj: T | Array<T>, prop: Extract<keyof T, string>, receiver: T) {
+  get<T extends {}>(
+    obj: T | Array<T>,
+    prop: Extract<keyof T, string> | SymbolProp,
```

Comment:
Symbol is not a string, we were passing the symbol, typescript just wasn't catching it.